manifest.php

```diff
@@ -50,7 +50,7 @@
     'label'       => 'QTI test model',
     'description' => 'TAO QTI test implementation',
     'license'     => 'GPL-2.0',
-    'version'     => '39.3.0',
+    'version'     => '39.3.1',
```

Comment:
```suggestion
    'version'     => '39.4.0',
```

It's not a fix, is it?
The PR's title has to be updated as well, the modification is a new feature.